docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -34,6 +34,8 @@ export DOCKER_BUILD_PLATFORM=linux/arm64
 export DOCKER_BUILD_ARCH=arm64
 export ALPINE_IMAGE=arm64v8/alpine:3.14
 export POSTGRES_IMAGE=arm64v8/postgres:13-alpine
+export NGINX_IMAGE=arm64v8/nginx:1.19-alpine
```

Comment:
FYI, the way this is specified will change with https://github.com/airbytehq/airbyte/pull/11450

Reply:
awesome! that change looks great

btw, @jrhizor have you ever actually run into the issue I mentioned in this PR's description?
Basically if you build for amd64, you end up with the amd64 versions of the `openjdk` and `nginx` images in your local docker cache, so if you try to build for arm64 next, you get build errors unless you untag those base images.

Reply:
I haven't actually run into that for some reason. I already have the arm64 version of `nginx:1.19-alpine` cached locally, but I'm not sure why.

Reply:
Cool, thanks for the feedback.  I actually haven't found anyone else that has run into this issue yet which is partially why I left this as a draft PR, but it is very possible that my own setup just has something quirky going on which is why I run into this problem.